src/irr/asset/CImageLoaderTGA.cpp

```diff
@@ -86,6 +86,9 @@ bool CImageLoaderTGA::isALoadableFileFormat(io::IReadFile* _file) const
 		return false;
 	}
 	
+	/*
+		I can't load anything because of bellow
```

Comment:
wait, why, what happened?

Reply:
Well, all of the images placed in `color_space_test` doesn't contain information about gamma. I was trying to generate it through GIMP, but I couldn't as well, so I decided to bypass it

Reply:
well sure, but leave the code for handling gamma, if there is no info about gamma then assume 2.233333333..............

Reply:
okay, I will patch it